common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4432,7 +4432,13 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Fully Qualified class name for concerete impl to be used to enforce queues"),
     JOBNAME_ENRICHER_CLASS("jobname.enricher.class",
         "org.apache.hadoop.hive.ql.propertymodifier.NoEnrichment",
-        "Fully qualified name of class to be used to enrich job name");
+        "Fully qualified name of class to be used to enrich job name"),
+
```

Comment:
Do we need hive server deployment? If yes can be avoided?

Reply:
Discussed. Currently we will got with this.